rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -81,70 +90,143 @@ typedef struct
     RotationBatch rotation_batch;
 }JointsDataBatch;
 
+typedef struct MetaDataInfo {
+public:
+    uint img_id = -1;
+    std::string img_name = "";
+    ImgSize img_size = {};
+} MetaDataInfo;
+
 struct MetaData
```

Comment:
The prev struct introduced has a different styling in terms of placement of the brackets. Please make both of them uniform